### ISSUE TITLE
NTR: Anti-beta-fodrin autoantibodies present (Lab: serology & immunological tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8731]
New term request Anti-beta-fodrin autoantibodies present (elthat_101030035241):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000475
- label: Anti-beta fodrin antibody positivity
- parents: [Autoimmune antibody positivity (HP:0030057)](https://hpo.jax.org/app/browse/term/HP:0030057)
- pmid: PMID:11673564
- def: The presence of autoantibodies (immunoglobulins) in the blood circulation that react against a component of beta-fodrin.
- comment: Fodrin, a heterodimer composed of alpha and beta subunits, is an abundant protein of eukaryotic cell membrane skeleton. The alpha/beta-fodrin dimers self-associate head-to-head into tetramers and serve as the basic structural element of the membrane skeleton. Individuals with Sjogren syndrome may display antibodies to fodrin.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8731
- synonyms:
